### PR TITLE
Fixed projecting subitems

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
No Changelog entry required.
@normanjaeckel Please review later. Thanks.

File: openslides/agenda/static/js/agenda/projector.js [hunk 1]
```diff
@@ -82,25 +82,27 @@ angular.module('OpenSlidesApp.agenda.projector', ['OpenSlidesApp.agenda'])
...
-                    item.parentCount = item.parentCount - parentCountDelta;
-                    $scope.items.push(item);
-                }
+                    return false;
+                };
+                _.forEach(tree, function (node) {
+                    var result = getRootNode(node);
+                    if (result) {
+                        $scope.rootItem = result.item;
+                        $scope.tree = result.children;
```
Comment (normanjaeckel) [inline openslides/agenda/static/js/agenda/projector.js]: Hm. Do you want to break the loop here?

Comment (emanuelschuetze) [reply openslides/agenda/static/js/agenda/projector.js]: @FinnStutzenstein ?

**Comment by normanjaeckel** [comment]
Ok.

**Pull request merged**